### PR TITLE
zip command had missing arguments. found also a tag naming bug.

### LINKED ISSUE #2: Build Releases seem to be way bigger than originals
https://github.com/VSCodium/vscodium/issues/597#issuecomment-870749135

### PR DESCRIPTION
**Pull request by tibeer**
closes #2

Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

File: build_macos_arm.sh [hunk 1]
```diff
@@ -6,20 +6,21 @@ tag_fork=$(git tag -l | tail -n 1)
...
   echo "### CREATING RELEASE ###"
   gh release create "${tag}" --notes "Added Darwin ARM64 binary for M1 Macs.
   Compiled on my M1 Macbook Air in just about 5 minutes 🥳"
-  zip -r VSCode-darwin-arm64/VSCodium.zip VSCode-darwin-arm64/VSCodium.app -x "*.DS_Store"
-  gh release upload "${tag}" VSCode-darwin-arm64/VSCodium.zip
+  zip -r -X -y vscodium/VSCode-darwin-arm64/VSCodium.zip vscodium/VSCode-darwin-arm64/VSCodium.app -x "*.DS_Store"
+  gh release upload "${tag}" vscodium/VSCode-darwin-arm64/VSCodium.zip
 
   echo "### INSTALLING ###"
   sudo cp -r VSCode-darwin-arm64/VSCodium.app /Applications/VSCodium.app
```
Comment (daiyam) [inline build_macos_arm.sh:26]: Shouldn't it be `vscodium/VSCode-darwin-arm64/VSCodium.app`?

Comment (tibeer) [reply build_macos_arm.sh:26]: I guess you mean line 23? You can unfortunately only push zip files to GitHub.

Comment (daiyam) [reply build_macos_arm.sh:26]: no, on line 26

Comment (tibeer) [reply build_macos_arm.sh:26]: Oh, right. Thanks for the hint! I'll fix that.

**Pull request merged**